src/pip/_internal/cli/main_parser.py

```diff
@@ -2,13 +2,17 @@
 """
 
 import os
+import shutil
+import subprocess
 import sys
-from typing import List, Tuple
+from typing import List, Optional, Tuple
 
+from pip._internal.build_env import _get_runnable_pip
```

Comment:
We should make this a public attribute in the module. :P

Reply:
Agreed. I wasn't sure whether it was also worth moving it to somewhere like `misc`, but we can leave that for now.